.github/workflows/build_infra_images_cache.yml

```diff
@@ -25,6 +25,7 @@ on:
     - 'master'
     paths:
     - 'dev/infra/Dockerfile'
+    - 'dev/infra/lint.Dockerfile'
```

Comment:
Do we have a better naming? `lint.Dockerfile` looks a bit odd to me - Dockerfile looks like an extension.

Reply:
`Dockerfile.<something>` or `<something>.Dockerfile` are remmomand by docker officially:
https://github.com/docker/docker.github.io/blob/master/_includes/guides/create-dockerfile.md

It could also be: `Dockerfile.lint`

I don't have much preference between `Dockerfile.lint` and `lint.Dockerfile`, all is ok for me.

Reply:
BTW,
- `Dockerfile.<something>` may also bring some misleading in some case: https://github.com/apache/spark/pull/35772#discussion_r822239491
- We also use [dockerfile.java17](https://github.com/apache/spark/blob/master/resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile.java17) in Spark

But I think `Dockerfile.int` is ok in here.

Reply:
> Dockerfile looks like an extension.

There seems to be nothing wrong with Dockerfile as an extension, I just saw intelliJ can identify `lint.Dockerfile` and highlight syntax, but `Dockerfile.lint` not. 😂